### PR TITLE
rabbitmq: update 0.25.2->0.29.0 & add acceptance tests

### PR DESCRIPTION
**Pull request by bastelfreak**
depends on https://github.com/voxpupuli/puppet-prometheus/pull/342

### CODE REVIEW

File: spec/acceptance/rabbitmq_exporter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+require 'spec_helper_acceptance'
+
+describe 'prometheus rabbitmq_exporter' do
+  it 'rabbitmq_exporter works idempotently with no errors' do
+    pp = 'include prometheus::rabbitmq_exporter'
+    # Run it twice and test for idempotency
```
Comment (othalla) [inline spec/acceptance/rabbitmq_exporter_spec.rb:6]: do we really need to add this comment as we have the `it`?

**Review by othalla — APPROVED** [review]

File: data/defaults.yaml [hunk 2]
```diff
@@ -163,7 +163,7 @@ prometheus::rabbitmq_exporter::group: 'rabbitmq-exporter'
 prometheus::rabbitmq_exporter::package_ensure: 'latest'
 prometheus::rabbitmq_exporter::package_name: 'rabbitmq_exporter'
 prometheus::rabbitmq_exporter::user: 'rabbitmq-exporter'
-prometheus::rabbitmq_exporter::version: '0.25.2'
+prometheus::rabbitmq_exporter::version: '1.0.0-RC6.1'
```
[the remark this answers is not in the document]
Comment (bastelfreak) [reply data/defaults.yaml]: changed it to the latest 0.29.0 release

File: spec/acceptance/rabbitmq_exporter_spec.rb (hunk 1 above)
Comment (bastelfreak) [reply spec/acceptance/rabbitmq_exporter_spec.rb:6]: no, it isn't required

**Comment by othalla** [comment]
@bastelfreak why did you add the need-help label? is there something i can do?

**Comment by bastelfreak** [comment]
@othalla I don't really know why it fails, do you want to look into it? Same applies for https://github.com/voxpupuli/puppet-prometheus/pull/350

**Pull request closed without merging**